src/gluonts/model/r_forecast/_predictor.py

```diff
@@ -91,6 +91,7 @@ def __init__(
         self._rpackages = rpackages
 
         this_dir = os.path.dirname(os.path.realpath(__file__))
+        this_dir = this_dir.replace("\\", "/") #for windows
```

Comment:
Couldn't we just pathlib here?

Reply:
I have already test it. Without this line, the R path is error for "\\" in it. I add this line to change it to unix style. It works.

Reply:
Oh I see, it's actually the other way around? R doesn't like the `\` so we have to go for `/`?

Reply:
I have already test it. Without this line, the R path is error for "\\\\" in it. I add this line to change it to unix style. It works.

Reply:
It could be the least modification of original codes. I may use other import module. But I didn' test it.